client/benches/tps/utils.rs

```diff
@@ -294,7 +294,7 @@ impl MeasurerUnit {
         let transfer_rose = TransferBox::new(
             asset_id(self.name),
             1_u32.to_value(),
-            asset_id(self.next_name),
+            account_id(self.next_name),
```

Comment:
Good catch. It was broken but we didn't spot it, because we don't routinely run benchmarks.